framework/src/modules/network/index.js

```diff
@@ -0,0 +1,54 @@
+const { config } = require('./defaults');
+const Network = require('./network');
+const BaseModule = require('../base_module');
+
+/* eslint-disable class-methods-use-this */
+
+/**
+ * Network module specification
+ *
+ * @namespace Framework.Modules
+ * @type {module.NetworkModule}
+ */
+module.exports = class NetworkModule extends BaseModule {
+	static get alias() {
+		return 'network';
+	}
+
+	static get info() {
+		return {
+			author: 'LiskHQ',
+			version: '0.1.0',
+			name: 'lisk-core-network',
+		};
+	}
+
+	static get defaults() {
+		return config;
+	}
+
+	get events() {
+		return ['bootstrap', 'remoteEvent'];
+	}
+
+	get actions() {
+		return {
+			request: async action => this.network.actions.request(action),
+			send: action => this.network.actions.send(action),
```

Comment:
I suggest we revisit the naming convention here to align it with what we have already for the framework and more readable. 

For `event` model:

```js
// node 1 - chain
channel.invoke('network:send', data);


// node 2 - chain 
channel.subscribe('network:receive', data=> { doSomethingWith(data) });
```

For `request` model, since its just re-routing to existing actions, it must use same naming convention as:

```js

// node 1 - chain

get actions() {
	myAction: async (action) => {

	}
}


// If invokded on same node 
const result = await channel.invoke('chain:myAction');


// If invoked from other node 2
const result = await channel.invoke('network:invoke', {remoteAction: 'chain:myAction'})
```

In short if we change `remoteEvent` event to `receive` and `request` to `invokeRemote ` that will make it more readable and easy for end user.

Reply:
Updated to match the framework
instead i changed to
`request` => `invoke`
`send` => `publish`
`remoteEvent` => `subscribe`

Reply:
Network sounds like a channel within the channel (although it is a channel to communicate with the network =P) with the above naming, maybe something like below to make it more readable, 

`request` => `requestRemote`
`send` => `publishToNetwork` or `sendToNetwork` or just `send`
`remoteEvent` => `receive`